evap/static/ts/src/copy-to-clipboard.ts

```diff
@@ -1,19 +1,3 @@
-function copyToClipboard(text: string) {
```

Comment:
Don't we still have the [usage in staff_template_form](https://github.com/e-valuation/EvaP/blob/70434ef3391e643cdc4ae4ad4bfd04c4147bdaa0/evap/staff/templates/staff_template_form.html#L47)?